styles/komodo/catppuccin.user.less

```
@@ -0,0 +1,151 @@
+/* ==UserStyle==
+@name Komodo Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/komodo
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/komodo
+@version 2025.10.30
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/komodo/catppuccin.user.less
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Akomodo
+@description Soothing pastel theme for komodo
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@import "https://userstyles.catppuccin.com/lib/lib.less";
+
+@-moz-document domain("komo.do") {
+
+  html.dark {
+    #catppuccin(@darkFlavor);
+  }
+  html.light {
+    #catppuccin(@lightFlavor);
+  }
+
+  #catppuccin(@flavor) {
+    #lib.palette();
+    #lib.defaults();
+
+    --background: #lib.hslify(@crust) [];
+    --foreground: #lib.hslify(@text) [];
+    --card: #lib.hslify(@base) [];
+    --card-foreground: #lib.hslify(@text) [];
+    --popover: #lib.hslify(@base) [];
+    --popover-foreground: #lib.hslify(@text) [];
+    --primary: #lib.hslify(@accent) [];
+    --primary-foreground: #lib.hslify(@base) [];
+    --secondary: #lib.hslify(@surface0) [];
+    --secondary-foreground: #lib.hslify(@text) [];
+    --muted: #lib.hslify(@mantle) [];
+    --muted-foreground: #lib.hslify(@subtext0) [];
+    --accent: #lib.hslify(@base) [];
+    --accent-foreground: #lib.hslify(@text) [];
+    --destructive: #lib.hslify(darken(@red, 20%)) [];
```

```suggestion
    --destructive: #lib.hslify(mix(@red, @base, 25%)) [];
```
I think this works a bit better, and doesn't make the colour very distorted on Latte